bin/pycbc_live

```diff
@@ -855,6 +861,12 @@ parser.add_argument('--snr-opt-timeout', type=int, default=400, metavar='SECONDS
                     help='Maximum allowed duration of followup process to maximize SNR')
 
 
+parser.add_argument('--enable-embright-has-massgap', action='store_true', default=False,
```

Comment:
Is this argument needed? Eg could the code do the HasMassGap calculation only if the embright-massgap-max arg is given? 
I guess the issue is that if massgap-max has a default of 5 then effectively the value is always given, so we can't switch on that.
So in practice we don't have to give the massgap-max option only the 'enable' one?